alphagov/paas-cf
# [#145695325] Update ruby version use in Dashboard

## What

A recent update of the ruby buildpack has removed the minor version that
was in use by the dashboard. The update now ignores the minor ruby
version.

The dependencies have also been updated as part of this work.

## How to review

On top of a current master in dev do the following, datadog creds are in the password store. I recommend using the prod creds for testing with as it it read only

1. Checkout this branch and cd into `tools/paas_dashboard`
1. `cf push --no-start paas-dashboard`
1. `cf set-env paas-dashboard DD_API_KEY <datadog_api_key>`
1. `cf set-env paas-dashboard DD_APP_KEY <datadog_app_key>`
1. `cf start paas-dashboard`
1. Visit https://pass-dashboard.environmentname.dev.cloudpipelineapps.digital/paas-overview and see the working dashboard.

## Who can review

Not @LeePorte
